manifests/csi/vsphere-csi-controller-ss.yaml

```diff
@@ -43,45 +43,26 @@ spec:
         - name: vsphere-csi-controller
           image: gcr.io/cloud-provider-vsphere/vsphere-csi:v0.1.1
           env:
-            - name: VSPHERE_KUBE_CONFIG
-              value: "*"
             - name: CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
             - name: X_CSI_MODE
               value: "controller"
             - name: X_CSI_SPEC_REQ_VALIDATION
               value: "false"
+            - name: GLOG_LEVEL
+              value: "2"
```

Comment:
is this the default log value? these manifests are the "easy button" way of doing deployment, they should be minimal and not explicit default values